beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -225,22 +324,94 @@ func (s *Server) SubmitAttestations(w http.ResponseWriter, r *http.Request) {
 			})
 		}
 
+		wantedEpoch := slots.ToEpoch(att.Data.Slot)
+		vals, err := s.HeadFetcher.HeadValidatorsIndices(ctx, wantedEpoch)
+		if err != nil {
+			failedBroadcasts = append(failedBroadcasts, strconv.Itoa(i))
+			continue
+		}
+
+		subnet := corehelpers.ComputeSubnetFromCommitteeAndSlot(uint64(len(vals)), att.Data.CommitteeIndex, att.Data.Slot)
```

Comment:
This is incorrect because `Data.CommitteeIndex` is always 0 post-Electra. `CommitteeBits` should be used instead. I think calling `GetCommitteeIndex()` will be the easiest.